src/serialization/tokenizer.hpp

```diff
@@ -85,7 +85,9 @@ struct token
 		DOLLAR = '$',
 	};
 
+	/** the token_type of the current value */
```

Comment:
Isn't this comment completely useless? What does it actually add to the line of code it's annotating? We can see it's a token_type, and what it's the type of is obvious, isn't it?